src/store/wizardSlice.ts

```diff
@@ -126,6 +126,11 @@ export const wizardSlice = createSlice({
       state.aws.source = action.payload;
     },
     changeGcpShareMethod: (state, action: PayloadAction<GcpShareMethod>) => {
+      if (action.payload === 'withInsights') {
+        // Reset 'withGoogle' options to initial state
+        state.gcp.accountType = 'google';
```

Comment:
More of a question - do we need to keep the accountType set to "google"?

Reply:
@lavocatt does your :+1: mean you agree it is a good question or that you agree we need to keep the accountType set to "google"? :sweat_smile: 

My thinking was that it is the default state. But I am open to changing it to `undefined` - that would help keep the store state cleaner as you mentioned and that will really help us reduce our bug surface area for the submitHandler (and more, I suspect). So let's try it!

Reply:
:+1: to: wondering the same ;)

Reply:
Everything gets set to `undefined` now!